css/main.css

```diff
@@ -78,3 +78,59 @@ h1 {
 .main_button:hover {
   background: #60d0d0;
 }
+
+.sect_content {
+  padding-top: 70px;
+}
+
+.sect_content .container {
+  display: flex;
+  flex-direction: row;
+}
+
+.feature {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 18px;
+}
+
+h6 {
+  font-size: 15px;
+  font-family: "Lato", sans-serif;
+  color: rgb(102, 46, 121);
+  font-weight: bold;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+  text-align: center;
+  margin: 18px 0;
+}
+
+.feature:nth-of-type(2n+1) h6 {

```

Comment:
Looks like a hack. It's a designer's fault that you can't center it properly.
Just remove this class

Reply:
Thanks to this fault I found many new features of CSS pseudo-selectors

Reply:
@probil, should I centerise all icons in relation to text?

Reply:
@liubich yes

Reply:
Done